plugins/module_utils/hmc_resource.py

```diff
@@ -518,3 +518,60 @@ def authenticateHMCs(self, remote_hmc, username=None, passwd=None, test=False):
                 self.OPT['MKAUTHKEYS']['-U'] + username +\
                 self.OPT['MKAUTHKEYS']['--PASSWD'] + passwd
         self.hmcconn.execute(mkauthcmd)
+
+    def accept_level(self, system_name):
+        updlic_cmd = self.CMD['UPDLIC'] +\
+            self.OPT['UPDLIC']['-M'] + system_name +\
+            self.OPT['UPDLIC']['-O']['ACCEPT']
+
+        return self.hmcconn.execute(updlic_cmd)
+
+    def update_managed_system(self, system_name, upgrade=False, repo='ibmwebsite', level='latest', remote_repo=None):
+        if upgrade:
+            update_upgrade_flags = self.OPT['UPDLIC']['-O']['UPGRADE']
+        else:
+            update_upgrade_flags = self.OPT['UPDLIC']['-O']['RETINSTACT']
+        # build command
+        updlic_cmd = self.CMD['UPDLIC'] +\
+            self.OPT['UPDLIC']['-M'] + system_name +\
+            update_upgrade_flags +\
+            self.OPT['UPDLIC']['-T']['SYS'] +\
+            self.OPT['UPDLIC']['-R'] + repo +\
+            self.OPT['UPDLIC']['-L'] + level
+        if remote_repo:
+            host = remote_repo['hostname']
+            if host:
+                updlic_cmd += self.OPT['UPDLIC']['-H'] + host
+            username = remote_repo['userid']
+            if username:
+                updlic_cmd += self.OPT['UPDLIC']['-U'] + username
+            passwd = remote_repo['passwd']
+            if passwd:
+                updlic_cmd += self.OPT['UPDLIC']['--PASSWD'] + passwd
+            directory = remote_repo['directory']
+            if directory:
```

Comment:
we can remove this if condition if we set this as required=True

Reply:
True